packages/platform/src/editor/plugins/toolbar/BlockFormatDropDown.tsx

```diff
@@ -59,10 +59,12 @@ export default function BlockFormatDropDown({
   editor,
   blockType,
   disabled = false,
+  closeMenu,
```

Comment:
I suggest renaming this to `onSelect` which lets users of this component know when it will be called. So in the calling component (BackslashMenu L47) the property would be `onSelect={closeMenu}` which reads well, i.e. when we select in BlockFormatDropDown it will close the BackslashMenu.